recipes/opsin/2.1.0/meta.yaml

```diff
@@ -6,7 +6,7 @@ package:
     name: opsin
     version: 2.1.0
 build:
-  number: 0
+  number: 1
```

Comment:
Can you add an extra->notes section, that explains how to use the opsin.sh wrapper?